core/src/main/scala/cats/Inject.scala

```diff
@@ -0,0 +1,47 @@
+package cats
+
+import cats.arrow.FunctionK
+import cats.data.Coproduct
+import cats.syntax.either._
```

Comment:
Thanks. I have a false memory of seeing a green build, so this would have sat for a while before I noticed.